docs/e2e/requirements.md

```diff
@@ -54,13 +54,3 @@ The following are required for running all e2e tests against the fake RP
 | `File` | `secrets/metrics-int.cert` | Geneva metrics client certificate |
 | `File` | `secrets/metrics-int.key` |  Geneva metrics client key |
 | `File` | `secrets/.dockerconfigjson` |  Docker config allowing to pull geneva images |
-
```

Comment:
There is also two File entries that you need to update in L50 and L51

Reply:
removed them